libs/core/langchain_core/runnables/graph.py

```diff
@@ -344,6 +343,7 @@ def draw_png(
 
     def draw_mermaid(
         self,
+        with_styles: bool = True,
```

Comment:
This is technically a breaking change, could we use `*` in this API to force calling by named argument?

Reply:
done